spec/active_record/oracle_enhanced/type/text_spec.rb

```diff
@@ -31,6 +31,10 @@ class ::TestEmployee < ActiveRecord::Base; end
     class ::Test2Employee < ActiveRecord::Base
       serialize :comments
     end
+    class ::TestSerializedHashEmployee < ActiveRecord::Base
```

Comment:
maybe this class name should be `Test2SerializedHashEmployee` because in `spec/active_record/oracle_enhanced/type/binary_spec.rb` there is already a class `TestSerializedHashEmployee`.

Reply:
I thought those classes were scoped to the `describe` block, so there shouldn't be any conflict. E.g., there's already a `TestEmployee` class defined in both `binary_spec.rb` and `text_spec.rb`.

Happy to make this change though, if that's the safer route.

Reply:
If the describe block scopes these anyhow, I don't see how it would. More likely to me, Ruby just reopens the class and changes it as it goes which seems messy to me.

Reply:
Yeah, you're right. I think I conflated this with my understanding of [helper methods in RSpec](https://rspec.info/features/3-13/rspec-core/helper-methods/arbitrary-methods/).

I also overlooked that the existing specs seem to be [dealing with this using `remove_const`](https://github.com/brianthoman/oracle-enhanced/blob/5f07e232028cb187b4301cde8155f58dae4f2a12/spec/active_record/oracle_enhanced/type/text_spec.rb#L52-L56) after all the specs have run. 

I've added a `remove_const` to clean up the new test class I added. Do you think that addresses the issue? If not, I'll finally give in and change the name 😅. Maybe to something like `TextSpecSerializedHashEmployee` so we don't have to keep track of what Test Employee number we're up to across files.